auth-web/src/views/auth/home/DecideBusinessView.vue

```diff
@@ -23,8 +22,8 @@
       </v-col>
       <!-- Image Column -->
       <v-col cols="12" md="6">
-        <a :href="learnMoreUrl" target="_blank">
-          <v-img src="../../../assets/img/Step1_DecideBusiness_x2.png" aspect-ratio="1.2" contain></v-img>
+        <a :href="selectorWizardUrl" target="_blank">
+          <v-img src="../../../assets/img/Step1_DecideBusinesswizard_x2.png" aspect-ratio="1.2" contain></v-img>
```

Comment:
Instead of the path working "up" from the current folder, can this be specified "top down", for example "/assets/img/.." ? (I'm not sure what this would look like exactly.)

I'm thinking it might be less fragile to specify the asset from the root rather than from wherever this module is..?

Reply:
Fixed. 
I'm not sure why it was a full path in the first place 🤷